src/__test__/edit.test.js

```diff
@@ -0,0 +1,23 @@
+const editTask = (taskContainer, task) => {
+  const editText = document.createElement('input');
+  editText.type = 'text';
+  editText.className = 'editText';
+  editText.value = task.textContent;
+  taskContainer.replaceChild(editText, task);
+  editText.value = 'new task';
+  taskContainer.replaceChild(task, editText);
+  task.textContent = editText.value;
+};
+
+test('test edit function', () => {
```

Comment:
Please make sure you group this test, even if it is an individual one, within a `describe()` method as it is a project requirement.

<img width="351" alt="Screen Shot 2022-07-14 at 8 45 42" src="https://user-images.githubusercontent.com/82426682/178997097-0930e237-6ee5-4c32-991c-77eed719ce70.png">